### ISSUE TITLE
`ImportError: Missing optional dependency 'tables'. Use pip or conda to install tables.`

### ISSUE THREAD

**Issue by Amyloida** [DeepLabCut/DeepLabCut#1445]
### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Bug description

Labeling of videos results in error
"ImportError: Missing optional dependency 'tables'. Use pip or conda to install tables."

I've used pip install and conda install to install tables. It's definitely installed. I've also followed suggestions mentioned in #587 but none of the solutions fixed the error. 
 

![Code error](https://user-images.githubusercontent.com/88189871/127643198-55b55bb4-13ae-4824-9e14-68c7ea8ca9a2.png)

### Operating System

Windows 10
numpy 1.19.5
tables  3.6.1

### DeepLabCut version

dlc version 2.2rc3

### DeepLabCut mode

multi animal

### Device type

GeForce GTX 1080   

### Relevant log output

```shell
ImportError: Missing optional dependency 'tables'. Use pip or conda to install tables.

(See screenshot above for Traceback)
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by jeylau** [comment]
Hi @Amyloida, could you activate your environment and show the output of `conda list`?

**Comment by MMathisLab** [comment]
@Amyloida can you also confirm you are using this conda file (note, it is new for rc2.3) https://github.com/DeepLabCut/DeepLabCut/tree/master/conda-environments 

Note, in the future please search for the exact error message, as you will see there are other issues, and it would be better to post under them, vs. opening a new issue, thanks! 

https://github.com/DeepLabCut/DeepLabCut/issues/587
https://github.com/DeepLabCut/DeepLabCut/issues/1434
https://github.com/DeepLabCut/DeepLabCut/issues/1368
https://github.com/DeepLabCut/DeepLabCut/issues/587

**Comment by Amyloida** [comment]
> Hi @Amyloida, could you activate your environment and show the output of `conda list`?
Hi @jeylau, this is the output of conda list: 

```
In [1]: conda list
# packages in environment at C:\Users\jrb21105\Anaconda3\envs\DEEPLABCUT:
#
# Name                    Version                   Build  Channel
absl-py                   0.13.0                   pypi_0    pypi
alabaster                 0.7.12             pyhd3eb1b0_0
appdirs                   1.4.4                      py_0
argh                      0.26.2                   py38_0
argon2-cffi               20.1.0           py38h294d835_2    conda-forge
astroid                   2.6.2            py38haa95532_0
astunparse                1.6.3                    pypi_0    pypi
async_generator           1.10                       py_0    conda-forge
atomicwrites              1.4.0                      py_0
attrs                     21.2.0             pyhd8ed1ab_0    conda-forge
autopep8                  1.5.6              pyhd3eb1b0_0
babel                     2.9.1              pyhd3eb1b0_0
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
bayesian-optimization     1.2.0                    pypi_0    pypi
bcrypt                    3.2.0            py38he774522_0
black                     19.10b0                    py_0
bleach                    3.3.0              pyh44b312d_0    conda-forge
blosc                     1.21.0               h0e60522_0    conda-forge
brotlipy                  0.7.0           py38h2bbff1b_1003
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2021.7.5             haa95532_1
cachetools                4.2.2                    pypi_0    pypi
certifi                   2021.5.30        py38haa95532_0
cffi                      1.14.6           py38hd8c33c5_0    conda-forge
chardet                   4.0.0           py38haa95532_1003
click                     8.0.1              pyhd3eb1b0_0
cloudpickle               1.6.0                      py_0
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
console_shortcut          0.1.1                         4
cryptography              3.4.7            py38h71e12ea_0
cudatoolkit               11.2.2               h933977f_8    conda-forge
cudnn                     8.2.1.32             h754d62a_0    conda-forge
cycler                    0.10.0                   pypi_0    pypi
cython                    0.29.23                  pypi_0    pypi
debugpy                   1.3.0            py38h885f38d_0    conda-forge
decorator                 4.4.2                    pypi_0    pypi
deeplabcut                2.2rc3                   pypi_0    pypi
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
diff-match-patch          20200713                   py_0
docutils                  0.17.1           py38haa95532_1
entrypoints               0.3             pyhd8ed1ab_1003    conda-forge
ffmpeg                    4.3.1                ha925a31_0    conda-forge
filterpy                  1.4.5                    pypi_0    pypi
flake8                    3.9.0              pyhd3eb1b0_0
flatbuffers               1.12                     pypi_0    pypi
future                    0.18.2                   py38_1
gast                      0.4.0                    pypi_0    pypi
google-auth               1.32.1                   pypi_0    pypi
google-auth-oauthlib      0.4.4                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.34.1                   pypi_0    pypi
h5py                      3.1.0                    pypi_0    pypi
hdf5                      1.10.6               h7ebc959_0
icc_rt                    2019.0.0             h0cc432a_1
icu                       68.1                 h0e60522_0    conda-forge
idna                      2.10               pyhd3eb1b0_0
imageio                   2.9.0                    pypi_0    pypi
imageio-ffmpeg            0.4.4                    pypi_0    pypi
imagesize                 1.2.0              pyhd3eb1b0_0
imgaug                    0.4.0                    pypi_0    pypi
importlib-metadata        4.6.1            py38haa244fe_0    conda-forge
intel-openmp              2021.3.0                 pypi_0    pypi
intervaltree              3.1.0                      py_0
ipykernel                 6.0.1            py38h43734a8_0    conda-forge
ipython                   7.25.0           py38h43734a8_1    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
ipywidgets                7.6.3              pyhd3deb0d_0    conda-forge
isort                     5.9.2              pyhd3eb1b0_0
jedi                      0.17.2           py38haa95532_1
jinja2                    3.0.1              pyhd8ed1ab_0    conda-forge
joblib                    1.0.1                    pypi_0    pypi
jpeg                      9d                   h8ffe710_0    conda-forge
jsonschema                3.2.0              pyhd8ed1ab_3    conda-forge
jupyter                   1.0.0            py38haa244fe_6    conda-forge
jupyter_client            6.1.12             pyhd8ed1ab_0    conda-forge
jupyter_console           6.4.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.7.1            py38haa244fe_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_widgets        1.0.0              pyhd8ed1ab_1    conda-forge
keras-nightly             2.5.0.dev2021032900          pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
keyring                   22.3.0           py38haa95532_0
kiwisolver                1.3.1                    pypi_0    pypi
krb5                      1.19.1               hbae68bd_0    conda-forge
lazy-object-proxy         1.6.0            py38h2bbff1b_0
libblas                   3.9.0                     9_mkl    conda-forge
libcblas                  3.9.0                     9_mkl    conda-forge
libclang                  11.1.0          default_h5c34c98_1    conda-forge
libcurl                   7.77.0               h789b8ee_0    conda-forge
liblapack                 3.9.0                     9_mkl    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libspatialindex           1.9.3                h6c2663c_0
libssh2                   1.9.0                h680486a_6    conda-forge
llvmlite                  0.34.0                   pypi_0    pypi
markdown                  3.3.4                    pypi_0    pypi
markupsafe                2.0.1            py38h294d835_0    conda-forge
matplotlib                3.1.3                    pypi_0    pypi
matplotlib-inline         0.1.2              pyhd8ed1ab_2    conda-forge
mccabe                    0.6.1                    py38_1
mistune                   0.8.4           py38h294d835_1004    conda-forge
mkl                       2021.2.0           hb70f87d_389    conda-forge
mock                      4.0.3            py38haa244fe_1    conda-forge
moviepy                   1.0.1                    pypi_0    pypi
msgpack                   1.0.2                    pypi_0    pypi
msgpack-numpy             0.4.7.1                  pypi_0    pypi
mypy_extensions           0.4.3                    py38_0
nb_conda                  2.2.1            py38haa244fe_4    conda-forge
nb_conda_kernels          2.3.1            py38haa244fe_0    conda-forge
nbclient                  0.5.3              pyhd8ed1ab_0    conda-forge
nbconvert                 6.1.0            py38haa244fe_0    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
nest-asyncio              1.5.1              pyhd8ed1ab_0    conda-forge
networkx                  2.5.1                    pypi_0    pypi
notebook                  6.4.0              pyha770c72_0    conda-forge
numba                     0.51.1                   pypi_0    pypi
numexpr                   2.7.3            py38h4c96930_0    conda-forge
numpy                     1.19.5                   pypi_0    pypi
numpydoc                  1.1.0              pyhd3eb1b0_1
oauthlib                  3.1.1                    pypi_0    pypi
opencv-python             4.5.3.56                 pypi_0    pypi
opencv-python-headless    3.4.9.33                 pypi_0    pypi
openssl                   1.1.1k               h2bbff1b_0
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 21.0               pyhd8ed1ab_0    conda-forge
pandas                    1.3.0                    pypi_0    pypi
pandoc                    2.14.0.3             h8ffe710_0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
paramiko                  2.7.2                      py_0
parso                     0.7.0                      py_0
pathspec                  0.7.0                      py_0
patsy                     0.5.1                    pypi_0    pypi
pexpect                   4.8.0              pyhd3eb1b0_3
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    8.3.1                    pypi_0    pypi
pip                       21.1.3             pyhd8ed1ab_0    conda-forge
pluggy                    0.13.1           py38haa95532_0
proglog                   0.1.9                    pypi_0    pypi
prometheus_client         0.11.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.19             pyha770c72_0    conda-forge
prompt_toolkit            3.0.19               hd8ed1ab_0    conda-forge
protobuf                  3.17.3                   pypi_0    pypi
psutil                    5.8.0            py38h2bbff1b_1
ptyprocess                0.7.0              pyhd3eb1b0_2
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycodestyle               2.6.0              pyhd3eb1b0_0
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pydocstyle                6.1.1              pyhd3eb1b0_0
pyflakes                  2.2.0              pyhd3eb1b0_0
pygments                  2.9.0              pyhd8ed1ab_0    conda-forge
pylint                    2.9.3            py38haa95532_1
pyls-black                0.4.6                hd3eb1b0_0
pyls-spyder               0.3.2              pyhd3eb1b0_0
pynacl                    1.4.0            py38h62dcd97_1
pyopenssl                 20.0.1             pyhd3eb1b0_1
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.12.3           py38haa244fe_7    conda-forge
pyqt-impl                 5.12.3           py38h885f38d_7    conda-forge
pyqt5-sip                 4.19.18          py38h885f38d_7    conda-forge
pyqtchart                 5.12             py38h885f38d_7    conda-forge
pyqtwebengine             5.12.1           py38h885f38d_7    conda-forge
pyrsistent                0.17.3           py38h294d835_2    conda-forge
pysocks                   1.7.1            py38haa95532_0
pytables                  3.6.1            py38h153c448_3    conda-forge
python                    3.8.10          h7840368_1_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python-jsonrpc-server     0.4.0                      py_0
python-language-server    0.36.2             pyhd3eb1b0_0
python_abi                3.8                      2_cp38    conda-forge
pytz                      2021.1             pyhd3eb1b0_0
pywavelets                1.1.1                    pypi_0    pypi
pywin32                   300              py38h294d835_0    conda-forge
pywin32-ctypes            0.2.0                 py38_1000
pywinpty                  1.1.3            py38hd3f51b4_0    conda-forge
pyyaml                    5.4.1                    pypi_0    pypi
pyzmq                     22.1.0           py38h09162b1_0    conda-forge
qdarkstyle                2.8.1                      py_0
qt                        5.12.9               h5909a2a_4    conda-forge
qtawesome                 1.0.2              pyhd3eb1b0_0
qtconsole                 5.1.1              pyhd8ed1ab_0    conda-forge
qtpy                      1.9.0                      py_0    conda-forge
regex                     2021.7.6         py38h2bbff1b_0
requests                  2.25.1             pyhd3eb1b0_0
requests-oauthlib         1.3.0                    pypi_0    pypi
rope                      0.19.0             pyhd3eb1b0_0
rsa                       4.7.2                    pypi_0    pypi
rtree                     0.9.7            py38h2eaa2aa_1
ruamel-yaml               0.17.10                  pypi_0    pypi
ruamel-yaml-clib          0.2.6                    pypi_0    pypi
scikit-image              0.18.2                   pypi_0    pypi
scikit-learn              0.24.2                   pypi_0    pypi
scipy                     1.7.0                    pypi_0    pypi
send2trash                1.7.1              pyhd8ed1ab_0    conda-forge
setuptools                49.6.0           py38haa244fe_3    conda-forge
shapely                   1.7.1                    pypi_0    pypi
six                       1.15.0                   pypi_0    pypi
snowballstemmer           2.1.0              pyhd3eb1b0_0
sortedcontainers          2.4.0              pyhd3eb1b0_0
sphinx                    4.0.2              pyhd3eb1b0_0
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0
sphinxcontrib-htmlhelp    2.0.0              pyhd3eb1b0_0
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0
sphinxcontrib-serializinghtml 1.1.5              pyhd3eb1b0_0
spyder                    4.2.5            py38haa95532_0
spyder-kernels            1.10.2           py38haa95532_0
sqlite                    3.36.0               h8ffe710_0    conda-forge
statsmodels               0.12.2                   pypi_0    pypi
tabulate                  0.8.9                    pypi_0    pypi
tbb                       2021.3.0             h2d74725_0    conda-forge
tensorboard               2.5.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.0                    pypi_0    pypi
tensorflow                2.5.0                    pypi_0    pypi
tensorflow-estimator      2.5.0                    pypi_0    pypi
tensorpack                0.9.8                    pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.10.1           py38haa244fe_0    conda-forge
testpath                  0.5.0              pyhd8ed1ab_0    conda-forge
textdistance              4.2.1              pyhd3eb1b0_0
tf-slim                   1.1.0                    pypi_0    pypi
threadpoolctl             2.2.0                    pypi_0    pypi
three-merge               0.1.1              pyhd3eb1b0_0
tifffile                  2021.7.2                 pypi_0    pypi
tk                        8.6.10               h8ffe710_1    conda-forge
toml                      0.10.2             pyhd3eb1b0_0
tornado                   6.1              py38h294d835_1    conda-forge
tqdm                      4.61.2                   pypi_0    pypi
traitlets                 5.0.5                      py_0    conda-forge
typed-ast                 1.4.3            py38h2bbff1b_1
typing-extensions         3.7.4.3                  pypi_0    pypi
ucrt                      10.0.20348.0         h57928b3_0    conda-forge
ujson                     4.0.2            py38hd77b12b_0
urllib3                   1.26.6             pyhd3eb1b0_1
vc                        14.2                 hb210afc_5    conda-forge
vs2015_runtime            14.29.30037          h902a5da_5    conda-forge
watchdog                  1.0.2            py38haa95532_1
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
werkzeug                  2.0.1                    pypi_0    pypi
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
widgetsnbextension        3.5.1            py38haa244fe_4    conda-forge
win_inet_pton             1.1.0            py38haa95532_0
wincertstore              0.2             py38haa244fe_1006    conda-forge
winpty                    0.4.3                         4    conda-forge
wrapt                     1.12.1           py38he774522_1
wxpython                  4.0.7.post2              pypi_0    pypi
yaml                      0.2.5                he774522_0
yapf                      0.31.0             pyhd3eb1b0_0
zeromq                    4.3.4                h0e60522_0    conda-forge
zipp                      3.5.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h62dcd97_1010    conda-forge
```

**Comment by MMathisLab** [comment]
hi @Amyloida seems it is conda installed in the environment; can you run: `pip3 install git+https://github.com/PyTables/PyTables` maybe for some reason that version is not working as expected...

**Comment by backyardbiomech** [comment]
The problem is mentioned and solved in the first entry in #1368. Your pytables was installed by conda, as shown in your conda list output. Existing environments can be fixed by removing that version, and installing from pip

```
conda uninstall pytables
pip install tables
```

This does not happen with new environments since the yaml was fixed, unpinning pytables from conda in #1400.

**Comment by Jason66661010** [comment]
ImportError: Missing optional dependency 'tables'.  Use pip or conda to install tables.
I ran these commands:

conda uninstall pytables
pip install tables

but the error still there.
how to fix it?
(deeplabcut[gui] version is 2.2.0.2)
